src/main/cpp/src/SparkResourceAdaptorJni.cpp

```diff
@@ -33,7 +33,6 @@ namespace {
 
 constexpr char const *RETRY_OOM_CLASS = "com/nvidia/spark/rapids/jni/RetryOOM";
 constexpr char const *SPLIT_AND_RETRY_OOM_CLASS = "com/nvidia/spark/rapids/jni/SplitAndRetryOOM";
-constexpr char const *CUDF_EXCEPTION_CLASS = "ai/rapids/cudf/CudfException";
 constexpr char const *JAVA_OOM_CLASS = "java/lang/OutOfMemoryError";
```

Comment:
while you are at it, this could be replaced with `OOM_CLASS`: https://github.com/rapidsai/cudf/blob/4c456cb15be5ff284aa441d3fdc972fe5c761fa7/java/src/main/native/include/jni_utils.hpp#L39

Reply:
That will be cudf work, I'll do it here: https://github.com/rapidsai/cudf/pull/13354